library/std/src/io/stdio.rs

```diff
@@ -992,6 +992,9 @@ pub fn set_output_capture(sink: Option<LocalStream>) -> Option<LocalStream> {
 /// the global stream.
 ///
 /// However, if the actual I/O causes an error, this function does panic.
+///
+/// Writing to non-blocking stdout/stderr can cause an error, resulting in
+/// this function to panic.
```

Comment:
```suggestion
/// Writing to non-blocking stdout/stderr can cause an error, causing this
/// function to panic.
```

Reply:
This will mean cause is repeated twice "cause an error, causing this.." which might not look good? I can change it to the suggestion "Writing to non-blocking stdout/stderr can cause an error, causing this function to panic" and in macros.rs if it still makes sense.

Reply:
Edited to avoid the repetition.

Reply:
Edited to avoid the repetition.

Reply:
Changed in latest commit